prober/derp.go

```diff
@@ -434,10 +440,7 @@ func derpProbeBandwidth(ctx context.Context, dm *tailcfg.DERPMap, from, to *tail
 		time.Sleep(100 * time.Millisecond) // pretty arbitrary
 	}
 
-	start := time.Now()
-	defer func() { transferTime.Add(time.Since(start).Seconds()) }()
-
-	if err := runDerpProbeNodePair(ctx, from, to, fromc, toc, size); err != nil {
+	if err := derpProbeBandwidthTUN(ctx, transferTime, from, to, fromc, toc, size); err != nil {
```

Comment:
can we put this behind a flag for now and maintain `derpProbeBandwidth()` as default? we may want to compare between the methods for a while

Reply:
If you want, I could have these run side-by-side and just use different metric names (or use a label to differentiate).

Reply:
Alternately, if we just want to let the non-TUN version run for a while, then deploy the TUN version, and are unlikely to switch back, we could just leave it as-is and just not deploy it for a while.

Reply:
just a flag to pick one or the other